src/mbed_os_tools/test/host_tests_runner/__init__.py

```diff
@@ -20,6 +20,6 @@
 
 """
 
-from setuptools_scm import get_version
+from pkg_resources import get_distribution
 
-__version__ = get_version(root='../../../..', relative_to=__file__)
+__version__ = get_distribution("mbed-os-tools").version
```

Comment:
Do we also need something similar in the `detect` package?

Reply:
`detect` package doesn't need version info at runtime.